app/scss/components/_buttons.scss

```diff
@@ -16,11 +16,10 @@
  */
 .button{
   display: inline-block;
-  padding: .8rem 1.5rem;
+  padding: .8rem 1.3rem;
   background: #888;
   color: $white;
   border: solid thin rgba($black, .1);
-  text-shadow: 0 -.05em .02em rgba($black, .5);
```

Comment:
@bursteinz why take the text-shadow away? Are we keeping it flat design? I think most buttons have at least a little text shadow even if not so obvious so maybe we could reduce the shadow size?